frontend/libs/studio-components-legacy/src/components/StudioCodelistEditor/StudioCodeListEditorRow/StudioCodeListEditorRow.tsx

```
@@ -2,24 +2,30 @@ import type { CodeListItem } from '../types/CodeListItem';
 type StudioCodeListEditorRowProps = {
   error: ValueError | null;
   item: CodeListItem;
   number: number;
-  onBlurTextResource: (newTextResource: TextResource) => void;
+  onBlurTextResource: (textResource: TextResource) => void;
   onChange: (newItem: CodeListItem) => void;
-  onChangeTextResource: (newTextResource: TextResource) => void;
+  onChangeTextResource: (textResource: TextResource) => void;
   onDeleteButtonClick: () => void;
   textResources: TextResource[];
+  onCreateTextResource?: ({}: CreateTextResourceInternalArgs) => void;
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)